napari/_qt/qt_dims.py

```diff
@@ -1,5 +1,11 @@
 from qtpy.QtCore import Qt, Signal
-from qtpy.QtWidgets import QWidget, QGridLayout, QSizePolicy, QScrollBar
+from qtpy.QtWidgets import (
+    QWidget,
+    QGridLayout,
```

Comment:
Here I'm not sure if we should be using a QGridLayout. I think we want to use a single top `QVBoxLayout` that contains a bunch of widgets with `QHBoxLayout`, one for each displayed axis. Each of these widgets with a `QHBoxLayout` can then have the label, and the slider (and then soon the play button after #607 goes in). Does that make sense?